GRAPHICS/N_0xC6730E0D14E50703.md

```diff
@@ -0,0 +1,17 @@
+---
+ns: GRAPHICS 
+---
+
+## _0xC6730E0D14E50703
+
+```c
+// 0xC6730E0D14E50703 
+void _0xC6730E0D14E50703(Any p0);
```

Comment:
Correct name would be `SET_PTFX_FORCE_VEHICLE_INTERIOR_FLAG` with BOOL `value` to instructs all effect instances to set a flag, considering them as part of the vehicle interior.